src/ui/preferences-window/tabs/AppearanceTab.swift

```diff
@@ -33,7 +33,6 @@ class AppearanceTab {
             LabelAndControl.makeLabelWithCheckbox(NSLocalizedString("Hide apps with no open window:", comment: ""), "hideWindowlessApps"),
         ])
         grid.column(at: 0).xPlacement = .trailing
-        grid.rowAlignment = .lastBaseline
```

Comment:
Could you please elaborate on what the issue was, and which macOS version / arch it was happening on?

Reply:
This line caused me to be unable to open the Appearance Tab without creating a crash.

In the crash log there were errors regarding the UI constraints. But no more details what exactly triggers this behaviour.

After that I had to debug for a while to get further here. I realised that `LabelAndControl` and this line are related.

After removing this line, there were no more crashes and the UI looked like before.

I suspect here is a deviation in Apple's UI API that if in `LabelAndControl` `makeLabelWithProvidedControl` leaves `labelPosition` on the default `.leftWithSeparator`, then there is probably a bug with the constraints.

Reply:
I tested with ad without, and I couldn't see any difference. So since it fixed it for you, let's bring that change forward.

Also, I'm wondering if it could actually help with https://github.com/lwouis/alt-tab-macos/issues/1062